src/util/mod.rs

```diff
@@ -49,6 +52,34 @@ pub mod worker;
 pub use self::rocksdb::properties;
 pub use self::rocksdb::stats as rocksdb_stats;
 
+static mut PANIC_MARK: AtomicBool = AtomicBool::new(false);
```

Comment:
I guess `mut` can be removed because `AtomicBool::store` and `AtomicBool::load` only takes `&self`. As a result, unsafe can be removed as well!

Reply:
Done